README.md

```diff
@@ -1,43 +1,17 @@
-# curriculum-databases-projects-template
+# Vet Clinic Database
 
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
 
 
-## Getting Started
-
-This repository includes files with plain SQL that can be used to recreate a database:
+This repository includes SQL files that can be used to set up and manage a database for a vet clinic:
 
 - Use [schema.sql](./schema.sql) to create all tables.
 - Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
-
-<a name="readme-top"></a>
-
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
+- Check [queries.sql](./queries.sql) for examples of queries that can be run on the database.
```

Comment:
- [ ] Please make sure that your `README` should be professional and it shouldn't contain any `cluttered` or `redundant`  information. Kindly remove it from here, because it is already added inside the `getting-started` section. 

![Screenshot from 2023-11-06 14-32-40](https://github.com/DevKaranJ/vet-clinic/assets/66476329/0e227690-8b0a-4eac-94f8-2ef92b781207)

- [ ] Kindly add the relevant information in the `tech-stack` section, because in this milestone you are not using either `React` or `express`. Adding the relevant information will help to understand the project requirements.

![image](https://github.com/DevKaranJ/vet-clinic/assets/66476329/5298f37c-fc4b-4344-a3cf-e0acd138141d)